### PR TITLE
Ep/add effnetb1 cutmix augmix sam e1 5avg 424x377

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/effnetb1_cutmix_augmix_sam_e1_5avg_424x377/requirements.txt
Comment (mike-ferguson) [inline brainscore_vision/models/effnetb1_cutmix_augmix_sam_e1_5avg_424x377/requirements.txt:1]: I think this is empty by accident. Should have: 

`torch`
`PIL`
`numpy`
`timm`
`albumentations`

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
requirements.txt is blank

**Comment by pellegreene** [comment]
Of note: Albumentations requires Python 3.8 or higher. As we are running on Python 3.7 for Travis and Jenkins the checks will not pass.

**Comment by pellegreene** [comment]
@mike-ferguson Should be good to merge if all looks good

**Comment by mike-ferguson** [comment]
God merged, see comment above regarding `albumentations`

**Review by mike-ferguson — APPROVED** [review]

**Pull request merged**